packages/braintree-integration/src/braintree-local-payment-methods/braintree-local-methods-payment-strategy.ts

```diff
@@ -114,6 +118,50 @@ export default class BraintreeLocalMethodsPaymentStrategy implements PaymentStra
 
         this.toggleLoadingIndicator(true);
 
+        if (this.isNonInstantPaymentMethod(payment.methodId)) {
```

Comment:
Do you like an idea to separate non instant payment method and default lpm execution logic? (you can create 2 different methods like executeWithNotInstantLPM and executeWithInstantLPM)

This change might improve execute method code readability

Reply:
Make sense. Need to take a look at how it will look like